plugins/woocommerce/lib/composer.json

```diff
@@ -28,10 +28,10 @@
 			"dep_namespace": "Automattic\\WooCommerce\\Vendor\\",
 			"dep_directory": "/packages/",
 			"packages": [
-				"league/container"
+				"league/container",
+			    "psr/container"
 			],
 			"excluded_packages": [
-				"psr/container"
 			],
```

Comment:
Not essential, but I think we could remove this block entirely (though it could be handy as a reminder that `excluded_packages` is a possible option, we aren't for instance including other unused options like `override_autoload`).

Reply:
Yes, I think that although this is not needed, it's useful to leave it in place for documentation purposes.